hkp4py/__init__.py

```diff
@@ -2,5 +2,5 @@
 Python HKP client module
 """
 
-from hkp4py.client import Key, Identity, KeyServer
-__all__ = ['Key', 'Identity', 'KeyServer']
+from vendor.hkp4py.client import Key, Identity, KeyServer, HTTPClientError
```

Comment:
Sorry is this a typo?

`vendor` should not be in here?

Reply:
should we also add a `HKP4PYBaseException`

Reply:
Ha, so that's what happens when you turn off eMail notifications...

Yes, "vendor." shouldn't be there. I missed that when reviewing the patch from my (vendored) copy of hkp4py.

I have no personal need for a BaseException ... my itch is sufficiently scratched if I can distinguish "Invalid key" from "Server is offline", basically.